blueprints/showbie-tooling/index.js

```diff
@@ -1,27 +1,29 @@
 /* eslint-env node */
 module.exports = {
-  description: 'Install Showbie code style enforcement configuration and tooling',
+  description:
+    'Install Showbie code style enforcement configuration and tooling',
 
   normalizeEntityName() {}, // no-op since we're just adding dependencies
 
   beforeInstall() {
+    // eslint-disable-next-line ember/named-functions-in-promises
     return this.addAddonsToProject({
       packages: [
         { name: 'ember-cli-eslint', target: '~3.0.0' },
-        { name: 'ember-cli-code-coverage', target: '~0.3.4' }
-      ]
-    })
-    .then(() => {
+        { name: 'ember-cli-code-coverage', target: '~0.3.4' },
```

Comment:
Why does this need to be split out? I don't think this will actually get installed properly; needs to be passed into `addAddonsToProject`: https://ember-cli.com/api/classes/Blueprint.html#method_addAddonsToProject.

Reply:
To be honest I'm not sure, I was just updating the methods that were already here.